autopush/db.py

```diff
@@ -437,6 +442,9 @@ def delete_all_for_user(self, uaid):
         chidmessageids = [x["chidmessageid"] for x in results]
         if chidmessageids:
             self.delete_messages(uaid, chidmessageids)
+        self.table.delete_item(uaid=uaid,
```

Comment:
Sadly, no. Because the ' ' record is special. There may be a way for me to add the ' '  to the batch, though...